client/src/services/deleteReview.js

```diff
@@ -1,11 +1,11 @@
-const deleteReview = async (userId, reviewId, reviewUserID) => {
+const deleteReview = async (reviewId, reviewUserID) => {
```

Comment:
I didn't notice this before, so my mistake. But we should not need to provide information from the front end about the Review object, like the review.userId - we can access that information in the back end router. The `req.params.id` will tell us which review to delete, from there we can find the `userId` for that review, and determine if the record should be deleted or not